### DISCUSSION TITLE
Non riesco a risolvere T(n)=3T(n/2) + O(nlogn) con metodo iterativo

### DISCUSSION THREAD

**Discussion by rimaout** [CS-Swap/Algoritmi-1#39] [Announcements]
[Esercizio originale](https://github.com/CS-Swap/Algoritmi-1/issues/6)

Ho provato a risolverlo in questo modo, ma sono bloccato
<img width="761" alt="image" src="https://github.com/CS-Swap/Algoritmi-1/assets/116072651/41f64350-0b1f-41e2-9eb8-22890da9955f">

**Comment by notedo** [comment]
![IMG_0363](https://github.com/CS-Swap/Algoritmi-1/assets/110790224/b454c3d2-2cf2-4136-b0b5-ff1a89a71d10)
